internal/service/chime/voice_connector_group_test.go

```diff
@@ -20,17 +20,17 @@ import (
 
 func TestAccChimeVoiceConnectorGroup_basic(t *testing.T) {
 	ctx := acctest.Context(t)
-	var voiceConnectorGroup *chime.VoiceConnectorGroup
+	var voiceConnectorGroup *chimesdkvoice.VoiceConnectorGroup
 
 	vcgName := sdkacctest.RandomWithPrefix(acctest.ResourcePrefix)
 	resourceName := "aws_chime_voice_connector_group.test"
 
-	resource.ParallelTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
```

Comment:
Were these meant to be moved into a `_serial` function?

Reply:
Yes - added them now